tidy3d/components/data/dataset.py

```diff
@@ -924,13 +976,16 @@ def box_clip(self, bounds: Bound) -> UnstructuredGridDataset:
 
         return self._from_vtk_obj(clean_clip)
 
-    @requires_vtk
     def interp(
         self,
         x: Union[float, ArrayLike],
         y: Union[float, ArrayLike],
         z: Union[float, ArrayLike],
-        fill_value: float = 0,
+        fill_value: Union[float, Literal["extrapolate"]] = "extrapolate",
```

Comment:
Perhaps we should warn that the default behavior is changing?

Reply:
True. Actually, after thinking about it more I decided to stick with our protocol here (no api breaks) even though `fill_value="extrapolate"` typically makes more sense and these classes were introduced just recently. I did add a warning though to let user know that this behavior will change. To do that, I assigned default value to `None` which is converted to 0 internally. This allows to understand whether `fill_value` was set explicitly or the default value is used.